packages/authenticator/amplify_authenticator/lib/src/utils/dial_code_options.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+
+class DialCodeOptions {
```

Comment:
Can we add comments here? You can copy over the comments mentioned above using `@template`/`@macro` tags